README.md

```diff
@@ -29,92 +31,86 @@ materials science, quantum chemistry, and biology.
 * [Citing DeepChem](#citing-deepchem)
 
 ## Requirements
-* [pandas](http://pandas.pydata.org/)
+
+DeepChem requires these packages on any condition.
+
 * [joblib](https://pypi.python.org/pypi/joblib)
-* [sklearn](https://github.com/scikit-learn/scikit-learn.git)
-* [numpy](https://store.continuum.io/cshop/anaconda/)
+* [pandas](http://pandas.pydata.org/)
+* [Pillow](https://pypi.org/project/Pillow/)
+* [scikit-learn](https://github.com/scikit-learn/scikit-learn.git)
 * [tensorflow](https://www.tensorflow.org/)
 
 ### Soft Requirements
 DeepChem has a number of "soft" requirements. These are packages which are needed for various submodules of DeepChem but not for the package as a whole.
 
 * [RDKit](http://www.rdkit.org/docs/Install.html)
-* [six](https://pypi.python.org/pypi/six)
 * [MDTraj](http://mdtraj.org/)
 * [PDBFixer](https://github.com/pandegroup/pdbfixer)
-* [Pillow](https://pillow.readthedocs.io/en/stable/)
 
-### Super easy install via pip3
+## Installation
 
-```bash
-pip3 install joblib pandas sklearn tensorflow pillow deepchem
-```
-
-### Easy Install via Conda
+### Install via conda (Recommendation)
 
 ```bash
-conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.3.0
+conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem
 ```
+
 If you want GPU support:
 ```bash
-conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem-gpu=2.3.0
+conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem-gpu
 ```
 
-**Note:** The above commands install the latest stable version of `deepchem` and _do not install from source_. If you need to install from source make sure you follow the steps [here](#using-a-conda-environment).
+### Install via pip (WIP)
 
-### Using a Docker Image
-Using a docker image requires an NVIDIA GPU.  If you do not have a GPU please follow the directions for [using a conda environment](#installing-from-source-in-a-conda-environment)
-In order to get GPU support you will have to use the [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) plugin.
-``` bash
-# This will the download the latest stable deepchem docker image into your images
-docker pull deepchemio/deepchem
+You are able to try to install deepchem via pip using the following command.  
+However, pip installation is under development, so this command may not work well. 
 
-# This will create a container out of our latest image with GPU support
-nvidia-docker run -i -t deepchemio/deepchem
-
-# You are now in a docker container whose python has deepchem installed
-# For example you can run our tox21 benchmark
-cd deepchem/examples
-python benchmark.py -d tox21
+`deepchem=>next_version`

```

Comment:
I should set the next deepchem version.
I think it is not necessary to support old version because the latest release(May 4, 2019) is too old.